app/scripts/dcmWalkingRetargeting.xml

```diff
@@ -40,7 +40,7 @@
 
   <module>
     <name>yarpdev</name>
-    <parameters> --device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice ovrheadset --sticks 0  --tfDevice transformClient --tfLocal /oculustf --tfRemote /transformServer --tf_left_hand_frame loculus --tf_right_hand_frame roculus --tf_root_frame oculusworld --stick_as_axis true --gui_elements 0</parameters>
+    <parameters> --device JoypadControlServer --use_separate_ports 1 --period 10 --name /joypadDevice/Oculus --subdevice ovrheadset --sticks 0  --tfDevice transformClient --tfLocal /oculustf --tfRemote /transformServer --tf_left_hand_frame loculus --tf_right_hand_frame roculus --tf_root_frame oculusworld --stick_as_axis true --gui_elements 0 --hfov 79 --left-pitch-offset -0.02 --left-yaw-offset 0.0125 --right-pitch-offset 0.0575 --right-yaw-offset -0.245</parameters>
```

Comment:
@S-Dafarra Can you explain a bit what are the effect of the changes from the default values?

Reply:
They fix the orientation of the cameras in the Oculus view. In this way, the images appear less overlapped. By specifying them, it will also be easier in the future to edit them.

Reply:
and what about the horizontal field of view `--hfov 79` which is changed from the default value of `105`?

Reply:
It is the "zoom", the effect of pressing "Z" multiple times.